app/config/services.yml

```diff
@@ -30,18 +30,17 @@ services:
         arguments:
             $emailSender: '%app.notifications.email_sender%'
 
+    # When your service definition only requires arguments to be specified,
+    # you can use a shorter syntax for these:
     AppBundle\Twig\AppExtension:
-        arguments:
-            $locales: '%app_locales%'
+        $locales: '%app_locales%'
 
-    AppBundle\EventListener\CommentNotificationSubscriber:
-        arguments:
-            $sender: '%app.notifications.email_sender%'
+    # The short syntax can also be inlined:
+    AppBundle\EventListener\CommentNotificationSubscriber: { $sender: '%app.notifications.email_sender%' }
```

Comment:
I'd prefer to be consistent and use the same syntax in the whole app, i.e.:
```
    AppBundle\EventListener\CommentNotificationSubscriber:
        $sender: '%app.notifications.email_sender%'
```

You explain the Yaml syntax here, which is not related to the Symfony Demo I think.